pulpcore/cli/common/generic.py

```diff
@@ -414,6 +414,7 @@ def resource_option(*args: Any, **kwargs: Any) -> Callable[[FC], FC]:
     context_table: Dict[str, Type[PulpEntityContext]] = kwargs.pop("context_table")
     capabilities: Optional[List[str]] = kwargs.pop("capabilities", None)
     href_pattern: Optional[str] = kwargs.pop("href_pattern", None)
+    expose_value: Optional[bool] = kwargs.get("expose_value", True)
```

Comment:
By specifying the default (`True`) we can be sure it _is_ a boolean.
```suggestion
    expose_value: bool = kwargs.get("expose_value", True)
```

Reply:
Also do we need to extract it here? Can we maybe just query `param.expose_value` in the callback?